contributing_to_docs/doc_guidelines.adoc

```diff
@@ -13,25 +13,21 @@ for guidelines for OpenShift / OKD 3.x documentation.
 
 toc::[]
 
-== Topic Metadata
-Every topic file should contain the following metadata at the top, with no line spacing in between, except where noted:
+== Topic/assembly metadata
+Every topic file should contain the following metadata at the top, with no line
```

Comment:
s/topic/assembly
(The mod docs guide says "We do not use the terms _topic_ or _topic-based documentation_ because they are too ambiguous.") The guidance for file metadata is different between assemblies and modules, so this is a little confusing.